iocp/data/patterns.ini

```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
 defang:		True
 
 [IP]
-pattern:	\b(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\b
+pattern:	\b((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$\b
```

Comment:
ok but need to remove the $ at the end in order to match IP in any part of the document and not only at the end of the line (example: 8.8.8.8|google.com)